debian/rules

```diff
@@ -57,10 +57,10 @@ override_dh_makeshlibs:
 # Create dbg packages.
 # FIXME: should we just have a rdma-core-dbg package ?
 override_dh_strip:
-	dh_strip -plibibcm1 --dbg-package=libibcm1-dbg
-	dh_strip -plibibumad3 --dbg-package=libibumad3-dbg
-	dh_strip -plibibverbs1 --dbg-package=libibverbs1-dbg
-	dh_strip -plibrdmacm1 --dbg-package=librdmacm1-dbg
+	dh_strip -plibibcm1 --dbgsym-migration='libibcm1-dbg (<< 12-1~)'
+	dh_strip -plibibumad3 --dbgsym-migration='libibumad3-dbg (<< 12-1~)'
+	dh_strip -plibibverbs1 --dbgsym-migration='libibverbs1-dbg (<< 12-1~)'
+	dh_strip -plibrdmacm1 --dbgsym-migration='librdmacm1-dbg (<< 12-1~)'
 
```

Comment:
Hi Jason, 
I'm trying to solve the issue without removing jessie, by adding if define statements in the debian/rules file.
 It is working, but I had to leave the debug packages in the debian/control file since the control file doesn't have option for ifdefs.
We can solve it by making the control file auto generated by creating it during the build time. 
Please let me know your thoughts on this, How important it is to support Jessie?